model/accounts_mgmt/v1/generic_notify_details_response_type.model

```diff
@@ -0,0 +1,41 @@
+/*
+Copyright (c) 2024 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// class that defines notify details response in general.
+class GenericNotifyDetailsResponse {
+	// Index of the returned page, where one corresponds to the first page. As this
+	// collection doesn't support paging the result will always be `1`.
+	Page Integer
+
+	// Number of items that will be contained in the returned page. As this collection
+	// doesn't support paging or searching the result will always be the total number of
+	// notify details.
+	Size Integer
```

Comment:
If this is not a searchable collection, why do we need page, size, and total? why can't we just have the Items field alone?

Reply:
Will be handled in https://issues.redhat.com/browse/OCM-5561